modules/ensemble/lib/widget/input/dropdown.dart

```diff
@@ -285,6 +289,9 @@ class SelectOneState extends FormFieldWidgetState<SelectOne>
         validatorKey.currentState!.validate();
       }
     });*/
+    widget.controller.textEditingController =
```

Comment:
why not just set value, why create a new textEditingController?

Reply:
Because to set the initial value of a text field, we must need to pass a `TextEditingController` so we cannot just update a value variable